.babelrc.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
       presets: [['@babel/preset-env', { targets: { node: '8' }, modules: 'commonjs' }]],
     },
     esmBrowser: {
-      presets: [['@babel/preset-env', { modules: false }]],
+      presets: [['@babel/preset-env', { targets: { ie: '11' }, modules: false }]],
```

Comment:
I just checked again and I think [this is not necessary](https://babeljs.io/docs/en/babel-preset-env#targets): 

> Sidenote, if no targets are specified, @babel/preset-env will transform all ECMAScript 2015+ code by default.

Reply:
Is explicit better than implicit? =)